setup.py

```diff
@@ -119,6 +119,13 @@ def run(self):
         sdist.run(self)
 
 
+efilter_version = __version__
+if "bdist_msi" in sys.argv and efilter_version[1] == "!":
+  # bdist_msi does not support a setuptools epoch version so we remove it
+  # as a work around.
+  _, _, efilter_version = efilter_version.partition("!")
```

Comment:
> http://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=bdist_msi

Still supported.

> The downside is that you might expect bug reports for something which is not our responsibility to support and that noone will use.

IMO if you use it, you make sure it works if that means filing a bug upstream or disable it and explain why.